modules/ibi-extra-manifests-configmap.adoc

```diff
@@ -83,6 +83,11 @@ $ oc create configmap sr-iov-extra-manifest --from-file=sriov-extra-manifest.yam
 ----
 configmap/sr-iov-extra-manifest created
 ----
++
+[NOTE]
+====
+If you add more than one extra manifest, and the manifests must be applied in a specific order, you must prefix the filenames of the manifests with numbers representing the order in which they should be applied. For example, `00-namespace.yaml`, `01-sriov-extra-manifest.yaml`, and so on.
```

Comment:
```suggestion
If you add more than one manifest, to apply the manifests in a specific order, you must prefix the filenames of the manifests with numbers in the order you want to apply them. For example, `00-namespace.yaml`, `01-sriov-extra-manifest.yaml`, and so on.
```

- Suggested a change to avoid should. See: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s
- Suggested to _replace/passive-voice/active-voice_